app/library/postgres.py

```diff
@@ -224,6 +224,25 @@ async def get_scenario_by_friendly_name_from_postgres(
 
 
 async def delete_scenario_from_postgres(scenario_id: str, session):
+    # Get scene first
+    scenario_obj: Scenario = get_scenario(scenario_id, session)
+    if scenario_obj is None:
+        raise ValueError("Scenario ID not valid")
+
+    # Delete the screenshot of scenario, if it has one
+    if scenario_obj.screenshot_url:
+        # Only delete from AWS if config is enabled for AWS support
+        if "aws" in config.get("app-env"):
+            s3_client = boto3.client(
+                "s3",
+                endpoint_url=f"https://s3.{config.get('s3.region')}.amazonaws.com",
+                region_name=config.get("s3.region"),
+            )
+            s3_client.delete_object(
+                Bucket=config.get("s3.bucket_name"),
+                Key=scenario_obj.screenshot_url,
```

Comment:
This won't work, the screenshot_url being saved is the full URL, and what you need to delete is only the key of the object. Try looking at `scene` table's screenshot_url vs the `scenario`:
`scene`: images/247bca4e5f3946619e2633243d9f90f2.jpeg
`scenario`: https://dcc-bp-public-dev.s3.us-east-2.amazonaws.com/assets/cd231a6d3603410a908633c01c7065a5

Reply:
Yeah I noticed in the scene logic it was appending the asset prefix on the backend and not frontend. I guess this is why. Will fix that.

Reply:
Yeah, when I implemented that endpoint initially (the screenshot), FE didn't have that asset prefix variable. Now it can be prepended in either FE or BE.

Reply:
Properly set now :)
![image](https://user-images.githubusercontent.com/13268990/126877825-911a384a-e340-43ec-9fae-1a85a8cb56c3.png)